interfaces/cython/cantera/onedim.pyx

```diff
@@ -1,7 +1,7 @@
 # This file is part of Cantera. See License.txt in the top-level directory or
 # at http://www.cantera.org/license.txt for license and copyright information.
 
-import interrupts
+from .interrupts import no_op
```

Comment:
The more directly equivalent way to write this would be `from . import interrupts`, right?

Reply:
Yes, I think so, although since its just one function I didn't see the need to keep the namespace. Since that's the only function in that module and that's the only place its used, I thought about removing the module entirely.

Reply:
I guess it's the balance between just one function and the fact that it's only used one time -- the  form you used certainly makes sense in the case where the function is used many times.

The reason for this module existing is written in the docstring for the `no_op` function: "a pure Python function is needed in order for KeyboardInterrupt events to be captured.". I'm not sure if there are any additional considerations regarding type annotations.

Reply:
So if the function were in the .pyx file, it would be compiled and couldn't be used as an interrupt, got it. Thanks!